src/base/bittorrent/sessionimpl.cpp

```diff
@@ -1270,11 +1272,12 @@ int SessionImpl::globalMaxInactiveSeedingMinutes() const
 
 void SessionImpl::setGlobalMaxInactiveSeedingMinutes(int minutes)
 {
-    minutes = std::max(minutes, -1);
+    minutes = std::max(minutes, BitTorrent::Torrent::NO_INACTIVE_SEEDING_TIME_LIMIT);
 
-    if (minutes != globalMaxInactiveSeedingMinutes())
-    {
-        m_globalMaxInactiveSeedingMinutes = minutes;
+    if (minutes != globalMaxInactiveSeedingMinutes()) {
+        m_globalMaxInactiveSeedingMinutes =
+                std::clamp(minutes, BitTorrent::Torrent::NO_INACTIVE_SEEDING_TIME_LIMIT,
+                           BitTorrent::Torrent::MAX_INACTIVE_SEEDING_TIME);
```

Comment:
```suggestion
        m_globalMaxInactiveSeedingMinutes = std::clamp(minutes
            , BitTorrent::Torrent::NO_INACTIVE_SEEDING_TIME_LIMIT, BitTorrent::Torrent::MAX_INACTIVE_SEEDING_TIME);
```

Reply:
Please recheck code format due to last changes for this PR. I can't find anything about line length in Coding Guidelines. Thank you.